.github/workflows/enso4igv.yml

```diff
@@ -148,7 +148,7 @@ jobs:
       - name: Build with Maven
         run: mvn -B -Pvsix package --file tools/enso4igv/pom.xml -Denso.parser.lib=`pwd`/enso_parser/ -DskipTests
 
-      - name: Archive NBM file
+      - name: Archive NBM files
```

Comment:
- the goal of this PR is to include _all necessary NBM files_ in the ZIP
- the [Enso IGV Plugin](https://github.com/enso-org/enso/actions/runs/18240840917) now contains two NBM files

<img width="1718" height="704" alt="image" src="https://github.com/user-attachments/assets/28b69e13-e547-4946-a0f1-dfac5d6b5a78" />